### DISCUSSION TITLE
圧縮方式に統一（compression method が圧縮方法、圧縮方式、圧縮の手法などの訳語にブレている）

### DISCUSSION THREAD

**Discussion by KenichiroTanaka** [pgsql-jp/jpug-doc#3344] [NGとする語]
### 重要度

低

### 問題点

compression method が圧縮方法、圧縮方式、圧縮の手法とブレている

https://github.com/pgsql-jp/jpug-doc/pull/2829

で指摘をいただきました。

### 貢献者として記載可否

以前の記載名を使用

**Comment by KenichiroTanaka** [comment]
個人的には「圧縮方式」が良い気がしますが、皆様どうでしょうか。

**Comment by tatsuo-ishii** [comment]
「圧縮方式」に私も一票。

**Comment by koizumistr** [comment]
「圧縮の手法」はちょっと、、、と思いますので、圧縮方法、圧縮方式どちらでも良いです。

**Comment by noborus** [comment]
私も圧縮方式で良いと思います。

**Comment by KenichiroTanaka** [comment]
皆様、ありがとうございます。圧縮方式で進めたいと思います。
どこかで時間を見つけてPRを出します。

**Comment by KenichiroTanaka** [comment]
https://github.com/pgsql-jp/jpug-doc/pull/2941
でPRしました

**Comment by koizumistr** [comment]
pg_dump.sgmlとpg_dumpall.sgmlに
「TOAST圧縮手法を設定するコマンドを出力しません。」
としてまだ「圧縮手法」が残っているようです。
ただ、これなんですが
「TOASTの圧縮方式を設定するコマンドを出力しません。」
とするのがいいのではないかと思います。lzとかzipに対抗する「TOAST圧縮手法」なるものがあるように読めてしまうので。
